### DISCUSSION TITLE
Creating a new Row in a different table using tb_CRUD:id

### DISCUSSION THREAD

**Discussion by insomniacj93** [Budibase/budibase#7635] [Help]
Hello,

So I've been trying to create a new row in a different table using the values stored already in the db.

The intention here is to enter the tb_example:id (edit screen) by clicking the "View" button and use the information in the repeaters as new input for a different table. Seems simple at first but the fact that the tb_example:id is a "Update" Form, once i get another form to copy over the values i get the "Truncated Double value" error for the ID. Is there any way i can workaround this issue?

Thank you in advance!

**Comment by melohagan** [comment]
Hey @insomniacj93 

Would you be able to provide some screenshots please? The app export may also be helpful to look at.

**Reply by insomniacj93** [reply]
@melohagan sorry again for late reply!

Managed to solve this by unselecting the Form on the save row action, basicly it was creating some conflict by having 2 Save Row actions taking information from the same Form.

Solution:
1- Unselect 'Form' on the save row action,
2- Manually add the inputs to each column on said Save Row.

![Screenshot 2022-11-15 162015](https://user-images.githubusercontent.com/107686004/201971784-4cd820fb-4a64-44ec-b123-0103565b244e.png)

**Reply by melohagan** [reply]
Thanks for sharing your solution!
